### ISSUE TITLE
Add Variable for Pull Request Comment 

### ISSUE THREAD

**Issue by derekln1** [tomasbjerre/pull-request-notifier-for-stash#26]
Hi Tomas,

Can you add a variable for pull request comment that we can filter by and also send with the http request?

Thanks,
Derek

**Comment by tomasbjerre** [comment]
yes =)

**Comment by derekln1** [comment]
Awesome.  What we are looking for is on a Comment added event, the new comment gets sent and used to trigger a special build.

Looking forward to this :)  Thanks!

**Comment by tomasbjerre** [comment]
I added it to the latest commit. Havent tested it yet but you should be able to add:
${PULL_REQUEST_COMMENT_TEXT}
to filter text and in filter regexp
.*
The variable will be empty if its not a "comment added" -event.

**Comment by derekln1** [comment]
Tomas, you're too fast!!! :)))
I was actually going to try this as a side project tonight to learn more of the SDK APIs and Java.  But i can still do that when I have time.

Your commit for this works.  I was able to build it and I confirmed that the comment filtering and sending via http request works.  For Jenkins it has issues with the comment if there are spaces.  Not a problem for me, I am going to use grep only for the keyword we need and use "-" or "_" for spaces.

Thanks again for the great support!  :)
